src-django/api/views.py

```diff
@@ -117,12 +120,64 @@ def get_queryset(self):
 
 
 class ConceptViewSet(viewsets.ModelViewSet):
+    CSV_COLUMN_MAPPING = {
+        "created": "created",
+        "last_modified": "modified",
+        "uuid": "uuid",
+        "name": "name",
+        "display_name": "display_name",
+        "description": "description",
+        "data_type": "datatype",
+        "mime_type": "mimetype",
+        "constraint": "constraint"
+    }
+
     model = models.Concept
     serializer_class = serializer.ConceptSerializer
 
     def get_queryset(self):
         return models.Concept.objects
 
+    @list_route(methods=['POST'])
+    def import_csv(self, request):
+        if 'csv' not in request.FILES:
+            return JsonResponse(
+                status=status.HTTP_400_BAD_REQUEST,
+                data={
+                    'success': False,
+                    'errors': ["Missing 'csv' parameter"]
+                }
+            )
+
+        csv = request.FILES['csv']
+
+        file_path = "/tmp/{0}.csv".format(str(uuid.uuid4()))
```

Comment:
shouldn't you check if the file exists or not first (even though it is statistically rare to happen)?